libraries/fc/CMakeLists.txt

```diff
@@ -47,6 +47,13 @@ IF(NOT "$ENV{OPENSSL_ROOT_DIR}" STREQUAL "")
   message(STATUS "Setting up OpenSSL root and include vars to ${OPENSSL_ROOT_DIR}, ${OPENSSL_INCLUDE_DIR}")
 ENDIF()
 
+IF( NOT OPENSSL_ROOT_DIR )
+   set( OPENSSL_ROOT_DIR /usr/local/opt/openssl CACHE FILEPATH "" )
```

Comment:
This should use the standard cmake functions like `find_path` to find appropriate paths based on hints and _fail_ if it does not find the path instead of setting an arbitrary path and expecting it to fail later if it is wrong.